index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <header class="menu-container">
+        <nav class="navbar">
+            <div>
+            <h1 class="logo">My Logo</h1>
+           </div>
+            <div>
+          <img src="./images/Union.png" alt="menu">
+            </div>
+        </nav>
+    </header>
+    <main class="main">
```

Comment:
- [ ] I would suggest rather than using `main` tag here please use `section` tag it is semantically correct as It represents a generic standalone section of the document, so use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info.